ai-ml/trainium-inferentia/examples/gradio-ui/gradio-app-stablediffusion.py

```diff
@@ -0,0 +1,33 @@
+import gradio as gr
+import requests
+import json
+from PIL import Image
+from io import BytesIO
+
+# Constants for model endpoint and service name
+model_endpoint = "/imagine"
+# service_name = "http://<REPLACE_ME_WITH_ELB_DNS_NAME>/serve"
```

Comment:
couldn't an environment variable be used here, and it can be passed through the container so users don't need to make any changes here? 

You can then do something like 

```suggestion
service_name = os.environ.get('SERVICE_ENDPOINT', 'http://localhost:8000')
```

When the `SERVICE_ENDPOINT` variable is present, that value will be used (i.e. - in the container when deployed on a cluster), otherwise the default value `http://localhost:8000` will be used (local setup, if that possible?)

Reply:
Yes, will do.